operations/node_management/Removing_a_Liquid-cooled_blade_from_a_System.md

```diff
@@ -45,7 +45,44 @@ This procedure will remove a liquid-cooled blades from an HPE Cray EX system.
     cray hsm inventory redfishEndpoints update --enabled false x9000c3s0b1
     ```
 
-### Step 3: Clear the node controller settings
+### Step 3: Clear out the existing Redfish event subscriptions from the BMCs on the blade
+
+1. (`ncn#`) Set the environment variable `SLOT` corresponding to the blades location:
+
+    ```bash
+    SLOT="x9000c3s0"
+    ```
+
+1. (`ncn#`) Clear the Redfish event subscriptions:
+
+    ```bash
+    for BMC in $(cray hsm inventory  redfishEndpoints list --type NodeBMC --format json | jq .RedfishEndpoints[].ID -r | grep $SLOT); do
+        PASSWD=$(cray scsd bmc creds list --targets $BMC --format json | jq .Targets[].Password -r)
+        SUBS=$(curl -sk -u root:$PASSWD https://${BMC}/redfish/v1/EventService/Subscriptions | jq -r '.Members[]."@odata.id"')
+        for SUB in $SUBS; do
+            echo "Deleting event subscription: https://${BMC}${SUB}" 
+            curl -i -sk -u root:$PASSWD -X DELETE https://${BMC}${SUB}
```

Comment:
Would be better to avoid putting the password on the curl command line, because that is visible in the output of `ps -ef` for any users logged into the system, regardless of access level.